doc/guide/Installation.ipynb

```diff
@@ -0,0 +1,138 @@
+    "\n",
+    "2. **Install dependencies**: Create the virtual environment *climada_env* with CLIMADA's dependencies:\n",
+    "```\n",
+    "    cd climada_python-x.y.z\n",
+    "    conda env create -f requirements/env_climada.yml --name climada_env\n",
+    "```\n",
+    "Finally, set the ``climada_python-x.y.z`` folder path into the environment using the following command:\n",
+    "```   \n",
+    "      source activate climada_env\n",
+    "      conda develop /your/path/to/climada_python-x.y.z/\n",
+    "      conda deactivate\n",
+    "```\n",
+    "\n",
+    "3. **Test installation**: Activate the environment, execute the installation tests and deactivate the environment when finished using CLIMADA::\n",
+    "```   \n",
+    "    source activate climada_env\n",
+    "    python3 tests_install.py\n",
+    "    source deactivate\n",
+    "```   \n",
+    "\n",
+    "   If the installation has been successful, an OK will appear at the end (the execution should last less than 2min).\n",
+    "\n",
+    "4. **Run tutorials**: Install and launch *jupyter notebook* in the same environment:\n",
+    "```   \n",
+    "    source activate climada_env\n",
+    "    conda install jupyter\n",
+    "    jupyter notebook --notebook-dir /path/to/climada_python-x.y.z\n",
+    "```   \n",
+    "\n",
+    "A browser window will show up. Open ``climada_python-x.y.z/doc/tutorial/1_main_CLIMADA.ipynb``. This is the tutorial which will guide you through all CLIMADA's functionalities. Execute each code cell to see the results, you might also edit the code cells before executing. See [tutorials](https://github.com/CLIMADA-project/climada_python/tree/main/doc/tutorial) for more information.\n",
+    "\n",
+    "## FAQs\n",
+    "* ModuleNotFoundError; CLIMADA libraries are not found. Try to include *climada_python-x.y.z* path in the environment *climada_env* path as suggested in Section 2 of `Install environment with Anaconda`. If it does not work you can always include the path manually before executing your code:\n",
+    "```   \n",
+    "    import sys\n",
+    "    sys.path.append('path/to/climada_python-x.y.z')\n",
+    "```   \n",
+    "\n",
+    "* ModuleNotFoundError; some python library is not found. It might happen that the pip dependencies of *env_climada.yml* (the ones specified after ``pip:``) have not been installed in the environment *climada_env*. You can then install them manually one by one as follows::\n",
+    "```   \n",
+    "    source activate climada_env\n",
+    "    pip install library_name\n",
+    "```   \n",
+    "\n",
+    "where ``library_name`` is the missing library.\n",
+    "\n",
+    "Another reason may be a recent update of the operating system (macOS).\n",
+    "In this case removing and reinstalling Anaconda will be required.\n",
+    "\n",
+    "* Conda right problems in macOS Mojave: try the solutions suggested here [https://github.com/conda/conda/issues/8440](https://github.com/conda/conda/issues/8440)\n",
```

Comment:
Conda right now has problems ...

Reply:
I didn't really understand what I should do with this comment? :)

Reply:
The sentence `"* Conda right problems in macOS Mojave:` is incomplete.

Reply:
I left it it's not really a sentence just a point, it doesn't sound wrong to me

Reply:
`Conda right problems` has no meaning...